src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
Please don't take me wrong. I'd like to discuss about that when you want. I just think it would be better doing it in a dedicated issue. I'll be opening one for that, if it hasn't been opened before, when I finish some tasks I'm currently doing.